homeassistant/components/uptimerobot/manifest.json

```diff
@@ -3,7 +3,7 @@
   "name": "UptimeRobot",
   "documentation": "https://www.home-assistant.io/integrations/uptimerobot",
   "requirements": [
-    "pyuptimerobot==21.11.0"
+    "pyuptimerobot==21.2.0"
```

Comment:
This PR is now a dependency bump and a new feature, please split that into 2 separate PR's